umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -74,6 +74,37 @@
         :Checksum nil,
         :MimeType nil}}))
 
+(def expected-distribution-related-url-record-CMR-5366
+  '({:URL "https://nsidc.org/daac/subscriptions.html",
```

Comment:
'({:URL "https://nsidc.org/daac/subscriptions.html",
     :URLContentType "DistributionURL",
     :Type "GET DATA,",
     :Subtype "Subscribe",
     :Description "Subscribe to have new data automatically sent when the data become available.,"}
    {:URL "https://n5eil01u.ecs.nsidc.org/MOST/MOD10A1.061/",
     :URLContentType "DistributionURL",
     :Type "GET DATA,",
     :Subtype "DIRECT DOWNLOAD",
     :Description "Direct download via HTTPS protocol.,"}
    {:URL "https://search.earthdata.nasa.gov/search?q=MOD10A1",
     :URLContentType "DistributionURL",
     :Type "GET DATA,",
     :Subtype "Earthdata Search",
     :Description "NASA's newest search and order tool for subsetting, reprojecting, and reformatting data.,"}
    {:URL "https://search.earthdata.nasa.gov/search?q=MOD10A1",
     :URLContentType "DistributionURL",
     :Type "GET DATA",
     :Subtype "Earthdata Search",
     :Description "NASA's newest search and order tool for subsetting, reprojecting, and reformatting data 2.",
     :Checksum "A12345F"}
    {:URL "https://search.earthdata.nasa.gov/search?q=MOD10A1",
     :URLContentType "DistributionURL",
     :Type "GET DATA",
     :Description "NASA's newest search and order tool for subsetting, reprojecting, and reformatting data 3."}))

Reply:
I added 2 more online resources from the CMR-5366.xml file above and added it here.  Also the description should not be a map, but just the description text itself.  I made a code suggestion above that would accomplish that.  This suggestion should replace the entire def function